### ISSUE TITLE
AppleALC: Stopped working on macOS Monterey(Conexant CX20724)

### ISSUE THREAD

**Issue by ggoraa** [acidanthera/bugtracker#1707]
Hello everyone
TL;DR: I installed macOS Monterey on my machine, and sound stopped working. I believe that this is not an issue with my configuration, because everything worked fine on Catalina, Big Sur, etc.
My sound codec is Conexant CX20724, I used 13 layout-id(I tried also 3, but also no luck)
Bootloader is obviously OpenCore, and all kexts are up to date.
OpenCore log: 
[opencore-2021-06-25-150249.txt](https://github.com/acidanthera/bugtracker/files/6717137/opencore-2021-06-25-150249.txt)
EFI: https://drive.google.com/file/d/1s3KeD1K8_eURoKd44mXxwsD0hVVqJCDY/view?usp=sharing

Btw, all kexts for AppleALC to work are injected(Lilu, AppleALC, AppleHDA)
![image](https://user-images.githubusercontent.com/61655029/123417878-0426fb00-d5c1-11eb-932b-075f2c20cdc3.png)

**Comment by khronokernel** [comment]
You've failed to enable beta OS support for Lilu or any of its plugins:

> <"darkwake=1 keepsyms=1 -v -no_compact_check -disablegfxfirmware alcid=13 -liludbg -wegdbg">

Additionally you are using release binaries of kexts in a developer beta OS, retest with [newer builds](https://dortania.github.io/builds/) that correctly support Monterey

Finally you've failed to provide any Lilu logs via `liludump=60`. Please do not run a first developer beta if you are unsure of how to maintain your own build or file reports. Please use help forums instead
